pallets/providers/README.md

```diff
@@ -378,3 +377,45 @@ Error thrown when trying to get a root from a Main Storage Provider without pass
 ### `SpRegisteredButDataNotFound`
 
 Error thrown when a user has a Storage Provider ID assigned to it but its metadata data does not exist in storage (storage inconsistency error, should never happen).
+
+## Slashing Protocol
```

Comment:
In general, I would write it not in the future ("will be slashed", "will accrue") but in the present tense ("are slashed", "accrues") considering as I understood this should be the docs of the actual slashing functionality.